package com.eolwral.osmonitor.core;

message processorInfo {
  required sint32 maxFrequency = 1;          /**< Maximum Frequency */
  required sint32 minFrequency = 2;          /**< Minimum Frequency */
  required sint32 maxScaling = 3;            /**< Maximum Scaling Frequency */
  required sint32 minScaling = 4;            /**< Minimum Scaling Frequency */
  required sint32 currentScaling = 5;        /**< Current Scaling Frequency */
  required string grovernors = 6;            /**< Current Grovernors */
  required uint32 number = 7;                /**< Number */
  required bool offLine = 8;                 /**< Off-line */
  required string avaiableGovernors = 9;	 /**< Available Grovernors */
  required string avaiableFrequeucy = 10;	 /**< Available Frequency */
}